libs/@blockprotocol/graph/src/shared/stdlib/subgraph/builder.ts

```diff
@@ -1,8 +1,14 @@
 import {
-  addDataTypesToSubgraphByMutation,
-  addEntitiesToSubgraphByMutation,
-  addEntityTypesToSubgraphByMutation,
-  addPropertyTypesToSubgraphByMutation,
+  resolveDataTypeEdgesInSubgraphByMutation,
+  resolveEntityEdgesInSubgraphByMutation,
+  resolveEntityTypeEdgesInSubgraphByMutation,
+  resolvePropertyTypeEdgesInSubgraphByMutation,
```

Comment:
Upon seeing these in usage perhaps we want to use `infer` instead of `resolve`?
`inferDataTypeEdgesInSubgraphByMutation`